### ISSUE TITLE
Growalone Source

### ISSUE THREAD

**Issue by Bexpg** [GrowtopiaNoobs/GrowtopiaServer#1259]
https://www.mediafire.com/file/null/GrowaloneOldEngine-master.zip/file
For build you need Vs 2010 c# express and  xna game studio 4.0

**Comment by DrOreo002** [comment]
1. Not an issue
2. Out of topic as hell, please stop

**Comment by Bexpg** [comment]
@iProgramMC  its issue right???

**Comment by iProgramMC** [comment]
> 1. Not an issue
> 2. Out of topic as hell, please stop

Correct. Also, that source is kinda crappy and out of date. You won't be able to do much with it.

**Comment by iProgramMC** [comment]
Hello, im from turkey im 17 years old and im coding c# c++ im like kidding with other skids. They think im kids but im just kidding with they and laughing.

**Comment by illiililillililiillii** [comment]
@Bexpg youre actually retard could you kill your self before u damage someone's brains and also dont post off topic shit here this is Private Server issues tab not fucking leaking platform

**Comment by iProgramMC** [comment]
@Bexpg go to leakeverythingretard.com if you wanna leak shit not here

**Comment by playingoDEERUX** [comment]
@Bexpg clown the source is literally publically available on @iProgramMC's github and for this behavior you will be banned from hosting your server properly for 1 week. (its going to crash by default now)

**Comment by iProgramMC** [comment]
agree

**Comment by xXAbood** [comment]
Hello, im from turkey im 17 years old and im coding c# c++ im like kidding with other skids. They think im kids but im just kidding with they and laughing.

**Comment by xXAbood** [comment]
> @Bexpg clown the source was literally publically available on @iProgramMC's github and for this behavior you will be banned from hosting your server properly for 1 week. (its going to crash by default now a couple times)

bruh i literally mass register like 5k accounts every 5 minutes yet he always tries to open back the server

**Comment by iProgramMC** [comment]
You also trying to crash? Glad me and @playingoDEERUX aren't the only ones,
along with @raiterjaki and @bykosws.

**Comment by iProgramMC** [comment]
It's kinda amazing how you **can** register 5 thousand accounts. That's weird.

**Comment by Bexpg** [comment]
I can show apache logs 0 ddos attack + no one create 5 accounts per second when people create accounts im getting logs : ) my server is down because im quited project 2 days before you can read announcements in my discord server

**Comment by Bexpg** [comment]
> https://www.mediafire.com/file/7oroz4rabkd9mbt/GrowaloneOldEngine-master.zip/file
> For build you need Vs 2010 c# express and xna game studio 4.0

 You can use picture... im own all copyright : )

**Comment by Bexpg** [comment]
@GrowtopiaNoobs  u have to ban @iProgramMC  he keep doing his trolling and he think his funny

**Comment by iProgramMC** [comment]
@Bexpg why ME exactly? I haven't been doing any trolling. You have.

**Comment by iProgramMC** [comment]
You're the clown here. The source is up on my github profile, stop releasing github projects and putting them on Mediafire just to gain attention - it won't work.

**Comment by playingoDEERUX** [comment]
bexpg will never understand. hes a kid and doesnt wanna accept the fact that hes a loser :)

**Comment by Bexpg** [comment]
> bexpg will never understand. hes a kid and doesnt wanna accept the fact that hes a loser :)

Prof.dr.playingodeerux is speaking

**Comment by Bexpg** [comment]
. hes a kid and doesnt wanna accept the fact that hes a loser :)

**Comment by Bexpg** [comment]
bexpg will never understand

**Comment by Bexpg** [comment]
you will never understand im best and im not loser

**Comment by Onlyin2** [comment]
> you will never understand im best and im not loser

You're the best? Yea that's why you're using leaked sources.

**Comment by Bexpg** [comment]
@Onlyin2  im using leaked source for security issues i keep my real project secret ill not release untill fix all bugs

**Comment by playingoDEERUX** [comment]
ur „real project“ is also a paste and thats such a lie like are u even serious??! I cant even take u serious anymore at all

**Comment by playingoDEERUX** [comment]
and 16 might still be a „teenager“ and not „adult“, i know but ur probably a lot younger and that is determinable by your behavior.
my opinion is your mental age is equal to 12.

**Comment by Bexpg** [comment]
my opinion is your mental age is equal to 12.

Prof.dr.playingodeeruxakaprohax0rDrdRdRdRrr

**Comment by Bexpg** [comment]
@iProgramMC  you was do  private ur source after i leaked u maked public

**Comment by Bexpg** [comment]
![20200322_043106](https://user-images.githubusercontent.com/61095632/77240348-0ba96880-6bf6-11ea-9be1-7b1f49b3125f.jpg)
Im quit ok lel

**Comment by Bexpg** [comment]
04:29

**Comment by cmdocmd** [comment]
@playingoDEERUX you are 16 and you still saying you are "teenager" you are an adult now

**Comment by Bexpg** [comment]
@cmdocmd  playingo age is 11 and I am sure playingo hasno responsibility, first you should take responsibility to be adult.

**Comment by iProgramMC** [comment]
> @cmdocmd playingo age is 11 and I am sure playingo hasno responsibility, first you should take responsibility to be adult.
@Bexpg you got a digit wrong it's actually 16 years old not 11.

**Comment by iProgramMC** [comment]
And 16 is not adult, 18+ is.

**Comment by Bexpg** [comment]
Playingo's age is  100 but brain age is 10  ;))))

**Comment by iProgramMC** [comment]
Both are wrong

**Comment by kuteshiki** [comment]
Hello, im from turkey im 17 years old and im coding c# c++ im like kidding with other skids. They think im kids but im just kidding with they and laughing.

**Comment by Bexpg** [comment]
@GrowtopiaNoobs  @kuteshiki  keep doing trolling And his age is 13 + i watched his youtube video he putting pencil into cat mouth LOL How gay cat fucker ill report the video to youtube

**Comment by Bexpg** [comment]
@kuteshiki  u probably from indognesia cat fucker

**Comment by Bexpg** [comment]
https://youtu.be/DDr3dzz2Cjk change video tittle to i fucked cat 2 minute Gay bitch

**Comment by Bexpg** [comment]
And look 0:06 He try make cat Blind LOl

**Comment by Bexpg** [comment]
@kuteshiki  why u putting pencil into cat eyes retard bitch

**Comment by DrOreo002** [comment]
> @kuteshiki u probably from indognesia cat fucker



> https://youtu.be/DDr3dzz2Cjk change video tittle to i fucked cat 2 minute Gay bitch



> And look 0:06 He try make cat Blind LOl



> @kuteshiki why u putting pencil into cat eyes retard bitch

Oh wow bro chill

**Comment by coldchip** [comment]
@Bexpg 
Talking about pencils, do you need me to draw and illustrate how your brain cells looks like? Nah nevermind, graphite would not bind with brain cells smaller that itself.

**Comment by Bexpg** [comment]
![image](https://user-images.githubusercontent.com/61095632/77251584-316b5780-6c60-11ea-83d7-08de0f6e22d7.png)

as i said she/he 12 yrs old

**Comment by Bexpg** [comment]
@kuteshiki  hi i'm a 12 yearold weaboo!!!

**Comment by Bexpg** [comment]
![image](https://user-images.githubusercontent.com/61095632/77251643-94f58500-6c60-11ea-9ca6-c7f1cdff1213.png)

**Comment by Bexpg** [comment]
Anyway, you're younger than me, I shouldn't underestimate you, but remember you have to respect more older than you

**Comment by playingoDEERUX** [comment]
what?? you know he might have a different culture where age doesnt really matter.
